src/System.Management.Automation/engine/Modules/PSModuleInfo.cs

```diff
@@ -890,7 +890,7 @@ internal void AddToCompatiblePSEditions(IEnumerable<string> psEditions)
         /// Describes whether the module was considered compatible at load time.
         /// Any module not on the System32 module path should have this as true.
         /// Modules loaded from the System32 module path will have this as true if they
-        /// have declared edition compatibility with PowerShell Core. Currently, this field
+        /// have declared edition compatibility with PowerShell. Currently, this field
```

Comment:
The feature is only for 6+.
```suggestion
        /// have declared edition compatibility with PowerShell 6+. Currently, this field
```